README.md

```diff
@@ -1 +1,15 @@
-Hello world
+# Status Core Dev Meetings
+
+### Previous Meetings
+ №  | Date | Notes | Recording |
+ ---|---|---|---|
+ 10 | Mon, January 14, 2019 at 13:00 UTC | [notes](notes/meeting10.md) | [video](https://www.youtube.com/watch?v=4vWaqa38Hrs)
+ 9  | Mon, December 17, 2018 at 13:00 UTC | [notes](notes/meeting9.md), [agenda](notes/meeting9-agenda.md) | [video](https://www.youtube.com/watch?v=HVylXrDBJHg)
```

Comment:
Would it be possible to add the agenda in-line at the top of the two notes? This way it doesn't depend on hackmd link still existing.